script/index.js

```diff
@@ -0,0 +1,43 @@
+const speaker = [
+  {
+    simage: './images/pic.png',
+    sName: 'Ntare G',
+    sLocation: 'Rwanda, Kigali',
+    sBio: 'A associate Software developer at microverse with  expertise in Full-stack development and DevOps.',
+  },
+  {
+    simage: './images/pic.png',
+    sName: 'Ntare Guy',
+    sLocation: 'Rwanda, Kigali',
+    sBio: 'A associate Software developer at microverse with  expertise in Full-stack development and DevOps.',
+  },
+  {
+    simage: './images/pic.png',
+    sName: 'Ntare Guy',
+    sLocation: 'Rwanda, Kigali',
+    sBio: 'A associate Software developer at microverse with  expertise in Full-stack development and DevOps.',
+  },
+  {
+    simage: './images/pic.png',
+    sName: 'Ntare Guy',
+    sLocation: 'Rwanda, Kigali',
+    sBio: 'A associate Software developer at microverse with  expertise in Full-stack development and DevOps.',
+  },
+];
+const speakerClass = document.querySelector('#speaker-id');
+
+for (let i = 0; i <= speaker.length; i += 1) {
+  speakerClass.innerHTML += `
+    <section class="speaker-card">
+                <div class="speaker-content">
+                    <img src='${speaker[i].simage}' width='150' alt="">
+                    <div class="speaker-desc">
```

Comment:
- [ ] Please kindly place an appropriate `alt` attribute on all `img` tags. do this everywhere else you have `img` tags